packages/shared/src/aws/tile.metadata.ts

```diff
@@ -14,4 +16,16 @@ export class TileMetadataTable extends TileMetadataTableBase {
         this.Imagery = new TileMetadataImagery(this);
         this.Provider = new TileMetadataProvider(this);
     }
+
+    /**
+     * Iterate over all records in the TileMetadataTable
+     */
+    async *[Symbol.asyncIterator](): AsyncGenerator<TileMetadataRecord, null, void> {
+        for await (const items of this.scan(Const.TileMetadata.TableName)) {
+            for (const row of items) {
+                yield DynamoDB.Converter.unmarshall(row) as any;
```

Comment:
Are there any records in here that are not `TileMetadataRecords` (Should we filter this some how)